promql/parser/functions.go

```diff
@@ -201,16 +226,32 @@ var Functions = map[string]*Function{
 		Variadic:   1,
 		ReturnType: ValueTypeVector,
 	},
+	"pi": {
+		Name:       "pi",
+		ArgTypes:   []ValueType{},
+		Variadic:   1,
```

Comment:
Pi() does not taje an optional parameter.

Reply:
Whoops didn't notice that when I copied it from another function.